pyannote/audio/pipelines/speaker_diarization.py

```diff
@@ -473,7 +478,10 @@ def apply(
 
         # exit early when no speaker is ever active
         if np.nanmax(count.data) == 0.0:
-            return Annotation(uri=file["uri"])
+            diarization = Annotation(uri=file["uri"])
+            if return_embeddings:
+                return {"diarization": diarization, "speaker_embeddings": {}}
```

Comment:
I'd rather return a tuple (_à la_ [`np.unique`](https://numpy.org/doc/stable/reference/generated/numpy.unique.html))

```suggestion
            if return_embeddings:
                return diarization, {}
```